api/TimeAddressableMediaStore.yaml

```diff
@@ -1888,11 +1888,7 @@ paths:
         Implementations need to handle situations where objects were uploaded but no flow segment was registered
         successfully.
 
-        The response may include PUT URLs for creating buckets for the media objects. These PUT URLs should
-        be used before uploading media objects. The object_id associated with each storage location has the
-        bucket name as its prefix.
-
-        The response may include PUT URLs for setting the CORS properties for the buckets and media objects.
+        The response may include PUT URLs for setting the CORS properties for the media objects.
```

Comment:
Should we call out removing this as an option too - I can't see anything in the docs to suggest it makes sense for S3 or Openstack at least.

Reply:
Agreed. Change made in https://github.com/bbc/tams/pull/148/commits/1492300e12b0505fe79e5a02d090dd56bd140f33